frontend/Dockerfile

```diff
@@ -10,6 +12,8 @@ RUN yarn install
 # add app
 COPY . /app/
 
+RUN echo "VITE_GRAPHQL_URL=${VITE_GRAPHQL_URL}" > /app/.env
```

Comment:
Not sure if that's the best way of doing that... I first tried to define a `volumes` in the `docker-compose.yaml` file and somehow trying to use it here but I wasn't able to make it work.

Reply:
Since we'll need this in the root, I think it's not a bad idea.